src/main/java/de/blackforestsolutions/dravelopsdatamodel/TravelPoint.java

```diff
@@ -1,32 +1,38 @@
 package de.blackforestsolutions.dravelopsdatamodel;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.hazelcast.nio.serialization.Portable;
+import com.hazelcast.nio.serialization.PortableReader;
+import com.hazelcast.nio.serialization.PortableWriter;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.experimental.Accessors;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.data.geo.Point;
 
+import java.io.IOException;
 import java.io.Serializable;
 import java.time.ZonedDateTime;
 
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.DravelOpsPortableFactory.DRAVEL_OPS_FACTORY_ID;
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.DravelOpsPortableFactory.TRAVEL_POINT_CLASS_ID;
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.classdefinition.TravelPointClassDefinition.*;
+
 @Getter
-@Slf4j
 @JsonDeserialize(builder = TravelPoint.TravelPointBuilder.class)
-public final class TravelPoint implements Serializable {
+public final class TravelPoint implements Serializable, Portable {
 
     private static final long serialVersionUID = 6106269076155338045L;
 
-    private final String name;
+    private String name;
 
-    private final Point point;
+    private Point point;
 
-    private final ZonedDateTime arrivalTime;
+    private ZonedDateTime arrivalTime;
 
-    private final ZonedDateTime departureTime;
+    private ZonedDateTime departureTime;
 
-    private final String platform;
+    private String platform;
```

Comment:
Maybe we need this to implement Portable, but we arent immutable anymore in this case. How do we enforce to not use setter for this method?

Reply:
Done